validphys2/src/validphys/checks.py

```diff
@@ -230,6 +230,45 @@ def check_pdfs_noband(pdfs, pdfs_noband):
     return {'pdfs_noband': pdfs_noband_combined}
 
 
+@make_argcheck
+def check_pdfs_plot_replicas(pdfs, pdfs_plot_replicas):
+    """Same as check_pdfs_noband, but for the pdfs_plot_replicas key.
+    Allows pdfs_plot_replicas to be specified as a list of PDF IDs or a list of
+    PDF indexes (starting from one)."""
+
+    msg = ("pdfs_plot_replicas should be a list of PDF IDs (strings) or a list of "
+           "PDF indexes (integers, starting from one)")
+    msg_range = ("At least one of your pdf_noband indexes is out of range. "
+                 "Note that pdf_noband indexing starts at 1, not 0.")
+
+    if pdfs_plot_replicas is None:
+        return
```

Comment:
```suggestion
        return {'pdfs_plot_replicas': []}
```

(this is in combination with the other comment below, you choose what to do)